exercicios/para-sala/exlogin.js

```diff
@@ -0,0 +1,20 @@
+let email = prompt ("Digite seu e-mail");
+console.log (email)
+let senha = prompt ("Digite sua senha");
+console.log (senha) 
+
+ if (email == "teste@reprograma.com" && senha == "teste123") {
+    alert ("acesso a página permitido");
+    console.log
+
+} else {
+   while (email != "teste@reprograma.com" || senha != "teste123") {
+    alert ("Ops! Você digitou algo errado. Digite email e senha novamente para acesso.")
```

Comment:
O exercício foi bem executado. Tem apenas dois pontos de observação: o console.log (mesmo comentário do outro arquivo) e em relação ao uso do if/else. Repare que no seu código existem duas mensagens de acesso permitido ao sistema, uma no if e outra no dentro do else também porém dentro do while. Porque o while também faz parte do else.
Lembre-se que o while já possui um if embutido (if com whey protein) que se a condição for verdadeira repete o código dentro do while. Desta forma não se faz necessário o uso do if else, apenas do while.

Uma outra observação é que com o alert de sucesso dentro do while, a mensagem de que o login foi feito com sucesso é emitida mesmo quando os dados estão inválidos, porque para estar dentro do while a condição dos dados estarem inválidos é verdadeira